File: src/test/java/th/ac/kmitl/it/soa/group10/backend/SpecifiedTaxRegistrationModelTest.java
package th.ac.kmitl.it.soa.group10.backend;

import org.junit.Test;
import th.ac.kmitl.it.soa.group10.backend.model.SpecifiedTaxRegistrationModel;

import static org.junit.Assert.assertEquals;

public class SpecifiedTaxRegistrationModelTest {

    @Test
    public void taxIDTest(){
        SpecifiedTaxRegistrationModel specifiedTaxRegistration = new SpecifiedTaxRegistrationModel();
        specifiedTaxRegistration.setTaxID("123768");
        assertEquals("123768", specifiedTaxRegistration.getTaxID());
    }

}

File: Backend/src/test/kotlin/th/ac/kmitl/it/soa/group10/tradeService/SpecifiedTaxRegistrationModelTest.kt
package th.ac.kmitl.it.soa.group10.tradeService

import th.ac.kmitl.it.soa.group10.tradeService.model.supplyChainTradeTransaction.SpecifiedTaxRegistrationModel
import org.junit.Assert.assertEquals
import org.junit.Test

class SpecifiedTaxRegistrationModelTest{
    var id = "10342135";

    @Test
    fun getDataTest(){
        var taxRegistration = SpecifiedTaxRegistrationModel(this.id);

        assertEquals(this.id, taxRegistration.id);

    }

}
